roles/custom/matrix-synapse/templates/goofys/systemd/matrix-goofys.service.j2

```diff
@@ -12,7 +12,11 @@ ExecStartPre=-{{ devture_systemd_docker_base_host_command_docker }} kill %n
 ExecStartPre=-{{ devture_systemd_docker_base_host_command_docker }} rm %n
 
 ExecStart={{ devture_systemd_docker_base_host_command_docker }} run --rm --name %n \
+			{% if matrix_synapse_log_driver %}
```

Comment:
This variable is not defined in defaults/main.yml and it probably should be. An if won't be necessary at all then